deafrica/data/copernicus_cci.py

```diff
@@ -62,7 +49,8 @@ def get_version_from_year(year: str) -> str:
     """
     try:
         year_num = int(year)
-        if year_num not in range(1992, 2020):
+        # Only allow from 1992 to 2020, inclusive
+        if year_num not in range(1992, 2021):
```

Comment:
The source https://cds.climate.copernicus.eu/cdsapp#!/dataset/satellite-land-cover currently only lists (supports?) dates up to 2020. Minor issue though, as I believe the available date range should change in a month or so anyway.

Reply:
range(1992,2021) finishes on 2020 anyway...